src/types.jl

```diff
@@ -169,14 +188,15 @@ mutable struct Solution
     x::Array{Float64, 1}
     y::Array{Float64, 1}
     s::Array{Float64, 1}
+    info::SCSInfo
     status::Symbol
     ret_val::Int
 
-    function Solution(x::Array{Float64, 1}, y::Array{Float64, 1}, s::Array{Float64, 1}, ret_val::Int)
+    function Solution(x::Array{Float64, 1}, y::Array{Float64, 1}, s::Array{Float64, 1}, info::SCSInfo, ret_val::Int)
```

Comment:
please update `status_map` (above) according to `https://github.com/cvxgrp/scs/blob/d3d1d4c49b0aede0ec491fe06c99aef5cfe598c7/include/glbopts.h#L18` (or similar in `MOI_wrapper.jl`)

I once opened a pull with this, but this was in pre-MOI times, so we left it for later, maybe now is the time;
maybe it's a good idea to share the definition with MOI?

Reply:
The status should be one of the MPB statuses. For custom status described by SCS, this should be `raw_status(info)`.
We should remove this from the struct since it is tied to MPB

Reply:
Ah no I see, we don't handle outside [-4, 1]. Let's leave it for a separate since I moved it to MPB_wrapper